sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -26,6 +27,8 @@
 import unittest
 from builtins import range
 
+import six
```

Comment:
We don't use `import six` any more in Beam (was systematically removed but I don't remember the reasoning). @tvalentyn might know more.

Reply:
`six` provides means to use some Python 2 syntactic idioms in Python 3. In Beam codebase we prefer to use idiomatic Python 3 as much as possible, and backport it to Python 2 using `python-future` if necessary.